Puzzlepart/did
# Fixing issue with navigating to a customer

### Your checklist for this pull request
 
### Review checklist

### Description
Fixing issue with navigating to a customer.

E.g. navigating to `https://[did]/customers/search/PZL`.

### How to test
1. Go to Timesheet (Overview tab)
2. Click on the customer name for one of your matched events
3. You should be sent to the customers page with the customer selected

### Related issues
Closes #791
